conf/views/unsuccessfulSubscription.njk

```diff
@@ -0,0 +1,23 @@
+{% extends "includes/layout.njk" %}
+
+{% from "macros/title.njk"                         import title %}
+
+{% block pageTitle %}
+  {{ title(messages("unsuccessfulSubscription.title")) }}
+{% endblock %}
+
+{% block mainContent %}
+
+  <div class="govuk-main-wrapper">
+    <div class="govuk-grid-row">
+      <div class="govuk-grid-column-two-thirds">
+
+        <h1 class="govuk-heading-xl">
+          {{ messages("unsuccessfulSubscription.heading") }}
+        </h1>
+
+      </div>
+    </div>
+  </div>
```

Comment:
Talk with the designers but this is probably more of a technical difficulties page